src/target/riscv/riscv-013.c

```diff
@@ -2837,6 +2837,12 @@ static int assert_reset(struct target *target)
 	return riscv013_invalidate_cached_progbuf(target);
 }
 
+static bool dcsr_ebreak_config_equals_reset_value(const struct target *target)
+{
+	RISCV_INFO(r);
+	return !(r->riscv_ebreakm || r->riscv_ebreaks || r->riscv_ebreaks);
```

Comment:
Duplicate `riscv_ebreaks`, should be `riscv_ebreaku`.

Reply:
Huge thanks for the catch!

Reply:
Addressed.